bookmarks/tests/test_bookmarks_api.py

```diff
@@ -92,6 +95,24 @@ def test_list_archived_bookmarks_should_filter_by_query(self):
         response = self.get(reverse('bookmarks:bookmark-archived') + '?q=#' + self.tag1.name, expected_status_code=status.HTTP_200_OK)
         self.assertBookmarkListEqual(response.data['results'], [self.archived_bookmark1])
 
+    def test_create_archived_bookmark(self):
```

Comment:
Let's also add a testcase to verify that bookmarks are not archived by default, in case the `is_archived` flag is not specified at all:
```python
    def test_create_bookmark_minimal_payload_does_not_archive(self):
        data = {'url': 'https://example.com/'}
        self.post(reverse('bookmarks:bookmark-list'), data, status.HTTP_201_CREATED)
        bookmark = Bookmark.objects.get(url=data['url'])
        self.assertFalse(bookmark.is_archived)
```